src/appengine/main.py

```diff
@@ -40,5 +40,9 @@
 except ImportError:
   pass
 
+from werkzeug.middleware.dispatcher import DispatcherMiddleware
 import server
-app = server.app
+import server_flask
+app = DispatcherMiddleware(server.app, {
```

Comment:
yes, I tried that but the problem comes when we use a prefix to go to a particular app lets say /jobs, and then enter server_flask then the "/jobs" part is removed from the path. I was not able to find a good solution to this. I will try to research more on this, if you find any particular implementation somewhere please share. There is not much available on this on internet.

Reply:
Is this because of how DispatcherMiddleware is implemented? IF so, we can simply implement our own matcher. There should be no technical reason why we can't pass the entire path as is without modifying it before it gets to the actual flask/webapp2 app.

Reply:
done, implemented a custom dispatcher middleware. This should work until the migration is completed.